src/api/contracts.ts

```diff
@@ -49,8 +49,11 @@ interface ContractsMapWProxy extends ContractsMap {
   TokenOWLProxy: DeployedContract,
 }
 
+// TODO: keep as 'development'
+const FE_CONDITIONAL_ENV = 'development'
```

Comment:
what for?

Reply:
testing without `npm run start:prod` and keeping logging